security/medusa/l2/kobject_process.c

```diff
@@ -30,7 +29,7 @@ medusa_answer_t process_kobj2kern(struct process_kobject *tk, struct task_struct
 
 		new_user = alloc_uid(tk->uid);
 		if (!new_user)
-			return MED_SKIP;
+			return MED_NO;
 		old_user = find_user(tsuid);
```

Comment:
This snippet comes from "Initial commit of Medusa Voyager to current kernel", so it makes original part of old Medusa system. I'm not sure about this change: MED_SKIP to MED_NO. In the case of alloc_uid() failure there may be MED_ERR. I really don't know why there was MED_SKIP!